.github/workflows/publish-github-release-pwm.yml

```
@@ -0,0 +1,23 @@
+name: Publish GitHub Release as newest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * 1-5' # Every hour on the hour on weekdays
+
+permissions:
+  contents: write
+  actions: read
+  id-token: write
+
+jobs:
+  publish-release-password-manager:
+    name: Publish Password Manager Release
+    uses: bitwarden/gh-actions/.github/workflows/_publish-mobile-github-release.yml@main
+    with:
+      workflow_name: "publish-github-release-pwm.yml"
```

⚒️  I think this  workflow should be renamed to `pm` rather than `pwm` 

```suggestion
      workflow_name: "publish-github-release-pm.yml"
```

---

Why on this workflow and not the android ones?